CHANGELOG-PENDING.md

```diff
@@ -6,12 +6,10 @@ Month, DD, YYYY
 
 ### BREAKING CHANGES
 
-- [go package] (Link to PR) Description @username
-[feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
-
 ### FEATURES
 
-- [go package] (Link to PR) Description @username
+- [feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
+- [feat(cmd|node): MutualPeers Node option and CLI flag #280](https://github.com/celestiaorg/celestia-node/pull/280) [@Wondertan](https://github.com/Wondertan)
```

Comment:
I suggest it should look smt like this: 
```suggestion
- [cmd](https://github.com/celestiaorg/celestia-node/pull/281) feat(cmd): give a birth to cel-shed and p2p key utilities #281 @Wondertan
- [cmd|node](https://github.com/celestiaorg/celestia-node/pull/280) feat(cmd|node): MutualPeers Node option and CLI flag @Wondertan
```

Reply:
Nope

Reply:
`Cmd` is a package and you are making a link to PR from the package word that doesn't make sense
Also, you remove a link to contributor

Reply:
Why? I was referring to Josh's example below in bug fixes section. 
Quote:
```
- [header] Added missing `err` value in ErrorW logging calls. @jbowen93
```

Reply:
I just want to clarify this for myself, as I've got confused when editing changelog-pending.md file for my race action PR 😅